Stratis.Bitcoin/Features/BlockStore/BlockStoreLoop.cs

```diff
@@ -14,6 +14,11 @@
 
 namespace Stratis.Bitcoin.Features.BlockStore
 {
+    /// <summary>
+    /// The BlockStoreLoop simultaneously finds and downloads blocks
+    /// and stores them in the BlockRepository
+    /// <see cref="DownloadAndStoreBlocks"/>
+    /// </summary>
     public sealed class BlockStoreLoop
     {
         internal readonly ConcurrentChain Chain;
```

Comment:
I fixed the summary for Chain.

I'm not sure that I always agree with commenting on EVERY property? I mean some of them fairly obivous as to what they are and what their purpose is. Seems a bit like overkill to me?